src/plugins/intel_gpu/include/intel_gpu/graph/program.hpp

```diff
@@ -146,6 +146,7 @@ struct program {
     std::vector<program_node*>& get_outputs() {
         return outputs;
     }  // ToDo: redesign reorder-inputs pass to make it const as_well as get_engine and get options
+    std::string get_output_layout_str();
```

Comment:
Seems redundant, do we need this?

Reply:
No, it is just for debug. removed it. thanks.